lib/util/error.js

```diff
@@ -497,6 +497,22 @@ shaka.util.Error.Code = {
    */
   'CONTENT_TRANSFORMATION_FAILED': 3019,
 
+  /**
+   * Important data is missing to be able to do the transmuxing of MSS.
```

Comment:
If this error is MSS-specific, the name of the error should probably include `MSS`.

Reply:
Added!